src/Nethermind/Nethermind.Network/P2P/Subprotocols/Eth/V65/PooledTxsRequestor.cs

```diff
@@ -7,6 +7,7 @@
 using Nethermind.Core.Caching;
 using Nethermind.Core.Collections;
 using Nethermind.Core.Crypto;
+using Nethermind.Network.P2P.Subprotocols.Eth.V62.Messages;
```

Comment:
this class should change namespace it is in `V65` now

Reply:
What do you mean "is in `V65` now"?
`using Nethermind.Network.P2P.Subprotocols.Eth.V62.Messages;` is needed to use const `TransactionsMessage.MaxPacketSize`, `TransactionsMessage` is a base message for transactions and is a part of `V62`. Peers are still sending to us pure `V62` `TransactionsMessages`. What more, when we are receiving `PooledTransactionsMessage`, we are handling it in `V62` handler, same as pure `V62` `Transactions`.

Reply:
`Nethermind.Network.P2P.Subprotocols.Eth.V65`, this class is used by multiple protocol versions, it is not in the right place.

Reply:
Ah, I got it. Yeah, it was initially used only by `eth65`, now `eth66` and `eth68` is using it as well. I will move it to general `Eth` namespace